.github/workflows/update-beats.yml

```diff
@@ -20,16 +19,20 @@ jobs:
       - uses: actions/setup-go@v3
         with:
           go-version-file: go.mod
-      - run: go version
-      - name: Setup Git
-        uses: elastic/apm-pipeline-library/.github/actions/setup-git@current
-      - name: Install Updatecli in the runner
-        uses: updatecli/updatecli-action@453502948b442d7b9a923de7b40cc7ce8628505c
-      - name: Run Updatecli
+      - uses: elastic/apm-pipeline-library/.github/actions/updatecli@current
         env:
+<<<<<<< Updated upstream
```

Comment:
There is a conflict

Reply:
https://github.com/elastic/apm-server/pull/10278/commits/f9ca67bfac2efca4ace4dd1005e16890a9f21cb3